src/Stryker.Core/Stryker.Core/Options/BooleanValidationExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using System.Linq;
+
+namespace Stryker.Core.Options
+{
+    public static class BooleanValidationExtensions
+    {
+        /// <summary>
+        /// Check if a nullable boolean is true or not null
+        /// </summary>
+        /// <param name="input">the boolean</param>
+        public static bool IsNotNullAndTrue(this bool? input)
```

Comment:
Just saying, as per my observations, when something like `bool?` pops up, it's usually a sign of missed enumeration :)